guidelines/index.html

```diff
@@ -443,15 +443,16 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
                					
                
                					
-               <p>In order to avoid confusion for implementers for whom backwards compatibility to WCAG 2 versions is important, new success criteria in WCAG 2.2 have been appended to the end of the set of success criteria within their guideline. This avoids the need to change the section number of success criteria from WCAG 2, which would be caused by inserting new success criteria between existing success criteria in the guideline, but it means success criteria in each guideline are no longer grouped by conformance level. 各ガイドライン内での達成基準の順番は、適合レベルに関する情報を暗示するものではなく、各達成基準自身の適合レベル表示 (A / AA / AAA) のみが適合レベルを示す。<a href="https://www.w3.org/WAI/WCAG22/quickref/">WCAG 2.2 クイックリファレンス</a>は、他の多数のフィルターや並べ替え方法と共に、適合レベルごとにグループ化された達成基準を見る方法を提供する。</p>
+               <p>WCAG 2 系のバージョンに対する後方互換性が重要となる開発者の混乱を招くことを避けるため、WCAG 2.2 における新しい達成基準はガイドライン内の一連の達成基準の末尾に追加されている。これにより、ガイドラインの既存の達成基準の間に新しい達成基準を挿入することによって生じるであろう、WCAG 2.1 からある達成基準のセクション番号の変更の必要性が、回避される。しかしながらこのことは、各ガイドライン内の達成基準がもはや適合レベルごとにグループ分けされていないことを意味する。各ガイドライン内での達成基準の順番は、適合レベルに関する情報を暗示するものではなく、各達成基準自身の適合レベル表示 (A / AA / AAA) のみが適合レベルを示す。<a href="https://www.w3.org/WAI/WCAG22/quickref/">WCAG 2.2 Quick Reference</a>は、他の多数のフィルターや並べ替え方法と共に、適合レベルごとにグループ化された達成基準を見る方法を提供するだろう。</p>
                				
             </section>
             				
             <section id="conformance-to-wcag-2-2"><div class="header-wrapper"><h4 id="conformance-to-wcag-2-2-0">WCAG 2.2 への適合</h4><a class="self-link" href="#conformance-to-wcag-2-2" aria-label="Permalink for this Section"></a></div>
                					
                
                					
-               <p>WCAG 2.2 uses the same conformance model as WCAG 2.0. It is intended that sites that conform to WCAG 2.2 also conform to WCAG 2.0 and WCAG 2.1, which means they meet the requirements of any policies that reference WCAG 2.0 or WCAG 2.1, while also better meeting the needs of users on the current Web.</p>
+               <p>WCAG 2.2 は WCAG 2.0 と同様の適合モデルを使用している。これは WCAG 2.2 に適合するサイトが WCAG 2.0 及び WCAG 2.1 にも適合することを意図している。つまり、それらのサイトが WCAG 2.0 又は WCAG 2.1 を参照するいかなる方針の要件も満たすと同時に、現在のウェブ上の利用者のニーズをより良く満たすことを意味します。</p>
```

Comment:
policyが「ポリシー」と「方針」で揺れている気がしますね…。
「いかなる」→「あらゆる」は趣味です

```suggestion
               <p>WCAG 2.2 は WCAG 2.0 と同様の適合モデルを使用している。これは WCAG 2.2 に適合するサイトが WCAG 2.0 及び WCAG 2.1 にも適合することを意図している。つまり、それらのサイトが WCAG 2.0 又は WCAG 2.1 を参照するあらゆる方針の要件も満たすと同時に、現在のウェブ上の利用者のニーズをより良く満たすことを意味する。</p>
```